### DISCUSSION TITLE
Triggering a flash notification from custom Livewire component

### DISCUSSION THREAD

**Discussion by duncanmcclean** [filamentphp/filament#8294] [Help] [answered]
Hello 👋🏻

I'm new to Filament so apologies if I've missed something in the documentation. I've built a custom page within Filament with it's own custom Livewire component. 

After the user clicks a button within my Livewire component, it triggers a method. At the end of that, I'd like to be able to trigger a flash notification for the user, letting them know something's happened successfully.

Is this something that's possible?

**Comment by danharrin** [comment, marked as answer]
Yes its possible, all of the examples on this page apply - https://filamentphp.com/docs/3.x/notifications/sending-notifications

You don't need to add `@livewire('notifications')` inside the panel builder though, we add it for you

**Reply by duncanmcclean** [reply]
That's perfect, thank you! I don't know what I must have been searching for me to _not_ see that page 😅

**Reply by danharrin** [reply]
make sure you're in the correct "package" of the docs, maybe?

**Reply by duncanmcclean** [reply]
Ah, that was probably it. Thanks!
